src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -294,6 +294,7 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   }, [isAsymRuneSide, isAsymAssetSide, virtualRuneFiatLiquidityAmount])
 
   const [slippageRune, setSlippageRune] = useState<string | undefined>()
+  const [isEstimateLoading, setIsEstimateLoading] = useState(false)
```

Comment:
Let's use slippage terminology, as fees / swep estimations are coming next and will also use `estimate` vernacular

Reply:
That's reasonable! https://github.com/shapeshift/web/pull/6111/commits/68cde5872ecd3295b818728dc2539b4df84d964c